book/tutorials/microwave/swesarr/swesarr_tut.ipynb

```diff
@@ -0,0 +1,72 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9fdd3e66",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# SWESARR"
```

Comment:
@db1950 - Note that you need a first cell that defines a header for your tutorial in order to show up in the table of contents.